content/en/_index.md

```diff
@@ -115,6 +134,14 @@ public static void main(String[] args) {
     exampleSnippet(projectId, filePath);
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+func main() {
+    // TODO(developer): Replace these variables before running the sample.
+    projectId := "my-project-id"
+    filePath := "path/to/image.png"
+    exampleSnippet(projectId, filePath)
```

Comment:
If we are requiring a main running I would much perfer to see errors handled here for Go.

Reply:
My understanding the runner function itself is not part of the sample, and this is really about how we document parameters in a way that keeps us from needing to set up samples in global scope.

Reply:
This was made a Java-only requirement. See https://github.com/GoogleCloudPlatform/samples-style-guide/pull/43#discussion_r839008433.